modules/social_features/social_group/modules/social_group_request/config/optional/block.block.membershiprequestsnotification_2.yml

```diff
@@ -0,0 +1,25 @@
+langcode: en
```

Comment:
What is the difference with the previous block? `social_group_request/config/optional/block.block.membershiprequestsnotification.yml`

Reply:
First block I use for socialblue theme. Second one for socialbase.

The same implementation I found in a some social features (example: block.block.booknavigation and block.block.booknavigation_2)

Reply:
Check!